### PR TITLE
Accept any price template in widget flow

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- Allow any price template / product in Widget flow

## Justify why they are needed

- This will make things much easier to develop and test

- We can consider reintroducing the check later and maybe log a warning to DD

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3544** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3544?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3545** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3545?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #3546** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3546?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
        * **PR #3547** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3546?utm_source=stack-comment).

File: apps/store/src/utils/PageLink.ts [hunk 1]
```diff
@@ -272,7 +272,7 @@ export const PageLink = {
         params.flow,
         params.shopSessionId,
         params.priceIntentId,
-        'calculate',
+        'calculate-price',
```
Comment (robinandeer) [inline apps/store/src/utils/PageLink.ts:275]: Fix route name!

**Review by alebedev — APPROVED** [review]
Good change.  Maybe a warning when non-widget template is used?  Could be a useful reminder for the future

**Review by guilhermespopolin — APPROVED** [review]
Well done :)

**Pull request merged**